app/models/ext_management_system.rb

```diff
@@ -27,7 +27,8 @@ def self.supported_types_and_descriptions_hash
 
   belongs_to :provider
   belongs_to :tenant
-
+  has_many :container_deployments_on, :class_name => 'ContainerDeployment', :inverse_of => :deployed_on_ems
+  has_many :container_deployments, :inverse_of => :deployed_ems
```

Comment:
The first relation is relevant to all ems, but the deployed_ems is only for ContainerManager, ill change it.